### PR TITLE
[assessment-tool] Small improvements.

### PR DESCRIPTION
**Pull request by ygorshenin**
* fixed references to icons for zoom in/zoom out
* line edit for query text and drop down for languages are replaced by QLabel
* fixed margin for sample view

@milchakov @bykoianko PTAL

### CODE REVIEW

**Comment by ygorshenin** [comment]
@bykoianko PTAL

File: search/search_quality/assessment_tool/sample_view.cpp [hunk 1]
```diff
@@ -157,14 +164,22 @@ void SampleView::EnableEditing(Edits & resultsEdits, Edits & nonFoundResultsEdit
...
+  m_query->hide();
+  m_langs->hide();
+
   m_showViewport->setEnabled(false);
   m_showPosition->setEnabled(false);
   ClearAllResults();
   OnSearchCompleted();
 }
 
+void SampleView::OnLocationChanged(Qt::DockWidgetArea area) {
```
Comment (bykoianko) [inline search/search_quality/assessment_tool/sample_view.cpp]: New line pls

Comment (ygorshenin) [inline search/search_quality/assessment_tool/sample_view.cpp]: Done

**Comment by ygorshenin** [comment]
Merge please.

**Pull request merged**